packages/devtools_app/lib/src/screens/app_size/app_size_table.dart

```diff
@@ -19,12 +19,12 @@ class AppSizeAnalysisTable extends StatelessWidget {
     required AppSizeController controller,
   }) {
     final treeColumn =
-        _NameColumn(currentRootLevel: rootNode.children[0].level);
+        _NameColumn(currentRootLevel: rootNode.level);
```

Comment:
We should use the root node instead of the first children (since we aren't supporting deferred loading yet)